ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1196,6 +1196,11 @@ func (r *RayClusterReconciler) calculateStatus(ctx context.Context, instance *ra
 	newInstance.Status.MinWorkerReplicas = utils.CalculateMinReplicas(newInstance)
 	newInstance.Status.MaxWorkerReplicas = utils.CalculateMaxReplicas(newInstance)
 
+	totalResources := utils.CalculateDesiredResources(newInstance)
```

Comment:
This function sums up the total resources in both head and workers regardless of the `rayStartParams` like `num-cpus: '0'`. In other words, it shows the total resources the CRD has not the Ray cluster processes themselves. Is this OK?

Reply:
This makes sense to me.